src/Microsoft.AspNet.Routing/RouteValueDictionary.cs

```diff
@@ -31,28 +31,32 @@ public RouteValueDictionary()
         }
 
         /// <summary>
-        /// Creates a RouteValueDictionary initialized with the provided input values.
+        /// Creates a <see cref="RouteValueDictionary"/> initialized with the provided input value.
         /// </summary>
-        /// <param name="values">Input values to copy into the dictionary.</param>
-        public RouteValueDictionary([NotNull] IDictionary<string, object> values)
-        {
-            _dictionary = new Dictionary<string, object>(values, StringComparer.OrdinalIgnoreCase);
-        }
-
-        /// <summary>
-        /// Creates a RouteValueDictionary initialized with the provided input values.
-        /// </summary>
-        /// <param name="values">Input values to copy into the dictionary.</param>
+        /// <param name="obj">An object to initialize the dictionary. The value can be of type
+        /// <see cref="IDictionary{TKey, TValue}"/> or <see cref="IReadOnlyDictionary{TKey, TValue}"/> or
+        /// any other object.
```

Comment:
We wanted to add it in the param description as remarks do not show up in intellisense.